### PR TITLE
Add missing error message and technique to resolve

### PR DESCRIPTION
**Pull request by ifly7charlie**
**What**: Documentation Improvement

**Why**: It's hard to find the solution to this error, but it is easy to resolve

**How**: Additional documentation including an example of solution

**Checklist**:

- [ x ] Documentation
- [ ] Tests
- [ x ] Ready to be merged
      

It would be good to document this in the configuration page as well

### CODE REVIEW

File: www/docs/errors.md [hunk 1]
```diff
@@ -77,6 +77,18 @@ The provider you tried to use failed when setting [PKCE or Proof Key for Code Ex
 The `code_verifier` is saved in a cookie called (by default) `__Secure-next-auth.pkce.code_verifier` which expires after 15 minutes.
 Check if `cookies.pkceCodeVerifier` is configured correctly. The default `code_challenge_method` is `"S256"`. This is currently not configurable to `"plain"`, as it is not recommended, and in most cases it is only supported for backward compatibility.
 
+#### ERR_JOSE_ALG_NOT_WHITELISTED
```
Comment (balazsorban44) [inline www/docs/errors.md:80]: I cannot seem to find a reference to this error code in our source code. Are you sure it is originated from `next-auth`?

**Review by balazsorban44 — CHANGES_REQUESTED** [review]
So I reviewed this, and tried looking for `"ERR_JOSE_ALG_NOT_WHITELISTED"` in our source code, but I don't think we actually use this. Do you maybe have a reproduction where the next-auth log actually sends you to: https://next-auth.js.org/errors#err_jose_alg_not_whitelisted?

Otherwise, this error documentation won't be seen.

File: www/docs/errors.md (hunk 1 above)
Comment (ifly7charlie) [reply www/docs/errors.md:80]: ok, that's embarrassing - correct fix, but I was generating custom JWT in the same method and got the error messages crossed. Actual one is ERR_JWK_KEY_SUPPORT and https://next-auth.js.org/errors#jwt_session_error 

[next-auth][error][jwt_session_error] 
https://next-auth.js.org/errors#jwt_session_error JWKKeySupport: the key does not support HS512 verify algorithm
    at check (/Users/melissa/Documents/GitHub/busyness/busyness/node_modules/jose/lib/jwa/index.js:45:11)
    at check (/Users/melissa/Documents/GitHub/busyness/busyness/node_modules/jose/lib/jwa/index.js:58:12)
    at jwsVerify (/Users/melissa/Documents/GitHub/busyness/busyness/node_modules/jose/lib/jws/verify.js:150:5)
    at Object.module.exports [as verify] (/Users/melissa/Documents/GitHub/busyness/busyness/node_modules/jose/lib/jwt/verify.js:235:5)
    at Object.<anonymous> (/Users/melissa/Documents/GitHub/busyness/busyness/node_modules/next-auth/dist/lib/jwt.js:105:30)
    at Generator.next (<anonymous>)
    at asyncGeneratorStep (/Users/melissa/Documents/GitHub/busyness/busyness/node_modules/next-auth/dist/lib/jwt.js:22:103)
    at _next (/Users/melissa/Documents/GitHub/busyness/busyness/node_modules/next-auth/dist/lib/jwt.js:24:194)
    at /Users/melissa/Documents/GitHub/busyness/busyness/node_modules/next-auth/dist/lib/jwt.js:24:364
    at new Promise (<anonymous>) {
  code: 'ERR_JWK_KEY_SUPPORT'
}

Comment (balazsorban44) [reply www/docs/errors.md]: so since the error is originated from jwt_session_error, it should be put under there, I think

**Review by balazsorban44 — APPROVED** [review]

**Pull request merged**